examples/transformers/MLFlow_X_HuggingFace_Finetune_a_text_classification_model.ipynb

```diff
@@ -0,0 +1,5244 @@
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "yRvLsqu4Ehu6"
+      },
+      "source": [
+        "While your training is ongoing, you can find this training in your dashboard. Log in to your [Databricks CE](https://community.cloud.databricks.com/) account, and click on top left to select machine learning in the drop down list. Then click on the experiment icon. See the screenshot below:\n",
+        "![landing page](https://drive.google.com/uc?export=view&id=1QxVaolr-L-w96pKUOiYQut3aSRE-04tC)\n",
+        "\n",
+        "After clicking the `Experiment` button, it will bring you to the experiment page, where you can find your runs. Clicking on the most recent experiment and run, you can find your metrics there, similar to:\n",
+        "![experiment page](https://drive.google.com/uc?export=view&id=1M-oycljsFAHBVip81Rprwx57Ape1uCq-)\n",
+        "\n",
+        "You can click on metrics to see the chart."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "M6CrKH4lMcUw"
+      },
+      "source": [
+        "## Evaluate the finetuned model\n",
+        "\n",
+        "Now we have finished the finetuning, let's see how it works. We will wrap the finetuned model into a HuggingFace `text-classification` pipeline. Using pipeline is optional, but it has several benefits including it automatically does tokenization and detokenization, so we can feed a string input and get a string output."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 12,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "eTrc2M6KEeNO",
+        "outputId": "c3ae8344-5086-4587-e233-cefe5927296e"
+      },
+      "outputs": [
+        {
+          "name": "stderr",
+          "output_type": "stream",
+          "text": [
+            "Xformers is not installed correctly. If you want to use memory_efficient_attention to accelerate training use the following command to install Xformers\n",
+            "pip install xformers.\n"
+          ]
+        },
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "Model prediction:  [{'label': 'spam', 'score': 0.999816358089447}]\n"
+          ]
+        }
+      ],
+      "source": [
+        "from transformers import pipeline\n",
+        "\n",
+        "pipe = pipeline(\n",
+        "    \"text-classification\",\n",
+        "    model=trainer.model,\n",
+        "    batch_size=8,\n",
+        "    tokenizer=tokenizer,\n",
+        "    device=0,\n",
+        ")\n",
+        "\n",
+        "# Make a random message.\n",
+        "sample_text = (\"WINNER!! As a valued network customer you have been selected \"\n",
+        "               \"to receivea £900 prize reward! To claim call 1234567. Claim \"\n",
+        "               \"code GEEZ. Valid 12 hours only.\")\n",
+        "print(\"Model prediction: \", pipe(sample_text))"
```

Comment:
I thought black would complain this isn't formatted, but looks like it doesn't. I'll take a look.

Reply:
formatting `.ipynb` files will be a bit tricky, since everything is expressed in strings. For reference, in Keras we only review and format the `.py` file, and auto generating the `.ipynb` file:
- .py file: https://github.com/keras-team/keras-io/blob/master/examples/nlp/fnet_classification_with_keras_nlp.py
- .ipynb file: https://github.com/keras-team/keras-io/blob/master/examples/nlp/ipynb/fnet_classification_with_keras_nlp.ipynb